lib/api/CDataFrameAnalyzer.cc

```diff
@@ -61,6 +65,20 @@ bool CDataFrameAnalyzer::handleRecord(const TStrVec& fieldNames, const TStrVec&
 
 void CDataFrameAnalyzer::run() {
     // TODO
+    auto outStream = m_OutStreamSupplier();
+    core::CRapidJsonConcurrentLineWriter outputWriter{*outStream};
+    for (std::size_t i = 0; i < m_AnalysisSpecification.rows(); ++i) {
+        outputWriter.StartObject();
+        outputWriter.String("id");
```

Comment:
It would be nice to define `"id"`, `"results"` and `"outlier_score"` as `std::string` constants.  This would also be more efficient because as it stands _every iteration of the loop_ is constructing temporary `std::string` objects from the `const char*`s to pass to `outputWriter.String()`.